src/script/conversation/ConversationRepository.ts

```diff
@@ -269,11 +270,16 @@ export class ConversationRepository {
     this.logger = getLogger('ConversationRepository');
 
     this.event_mapper = new EventMapper();
-    this.verificationStateHandler = new ConversationVerificationStateHandler(
+
+    // we register and store a handler, that we can manually trigger for incoming events from proteus and mixed conversations
+    this.proteusVerificationStateHandler = new ProteusConversationVerificationStateHandler(
       this.eventRepository,
       this.userState,
       this.conversationState,
     );
+    // we register a handler that will handle MLS conversations on its own
+    new MLSConversationVerificationStateHandler(this.eventRepository, this.conversationState, this.core);
```

Comment:
we dont want this to be saved to a method as well?

Reply:
Its not needed i think. This class has no public accessible methods that would need to be called. Its self sufficient